package.json

```diff
@@ -10,20 +10,24 @@
     "test-full": "npm run lint && npm run test"
   },
   "dependencies": {
+    "node-fetch": "^2.6.0",
     "react": "16.8.3",
     "react-native": "0.59.8",
     "react-redux": "^7.0.3",
-    "redux": "^4.0.1"
+    "redux": "^4.0.1",
+    "redux-saga": "^1.0.2"
   },
   "devDependencies": {
     "@babel/core": "7.4.4",
     "@babel/runtime": "7.4.4",
     "@types/jest": "24.0.13",
+    "@types/node-fetch": "^2.3.4",
```

Comment:
Just an FYI, React Native is not Node. It actually runs on JSCore, Apple's version of v8, so it's more of a web environment. So for proper types your going to need @types/react-native which I believe includes fetch.